app/presenters/courses/cohorts/students_presenter.rb

```diff
@@ -108,6 +124,12 @@ def total_students_count
     @total_students_count ||= scope.count
   end
 
+  def course_completed_students
+    @course_completed_students ||= scope.where.not(completed_at: nil)
+  end
+
+  delegate :count, to: :course_completed_students, prefix: true
```

Comment:
Why not just call `.count`?

Reply:
Initially, I had written a method named `course_completed_students_count` which returns the count of students who completed the course, but I changed to `delegate :count, to: :course_completed_students, prefix: true` to fix Rubocop warning 

As suggested by you, now I am calling `.count`

```erb
<p class="text-3xl font-bold"><%= @presenter.course_completed_students.count %></p>
```